dev_scripts_helpers/thin_client/build.py

```diff
@@ -98,7 +98,7 @@ def _main(parser: argparse.ArgumentParser) -> None:
         # dive_ver = run_command("dive --version")
         # _LOG.info("dive version=%s", dive_ver)
     # Linux specific updates.
-    if platform.system() == "Linux":
+    if hserver.is_external_linux():
```

Comment:
Nit: I'd make it mutually exclusive. A machine can't be Mac *and* Linux.

In that case, the comment should go inside the if-then-else